LICENSE.txt

```diff
@@ -0,0 +1,667 @@
+Package: uk.co.compucorp.civicaseextras
+Copyright (C) 2018, René Olivo <minet.ws@gmail.com>
```

Comment:
Compucorp Copyright info should be used. Please use the same from civicase repo.

Reply:
This was done automatically, thanks for spotting it.